tests/providers/snowflake/operators/test_snowflake_sql.py

```diff
@@ -163,7 +162,9 @@ def test_exec_success(sql, return_last, split_statement, hook_results, hook_desc
         )
 
 
-def test_execute_openlineage_events():
+@mock.patch("airflow.providers.openlineage.utils.utils.should_use_external_connection")
+def test_execute_openlineage_events(should_use_external_connection):
+    should_use_external_connection.return_value = False
```

Comment:
Do we have a test for `should_use_external_connection=True` now ?

Reply:
Snowflake should not have such test I think. However might add one for other DB, what do you think @kacpermuda ?

Reply:
Yes, something to append to a to do list.

Reply:
`test_execute_openlineage_events` in `test_redshift_sql` fulfills that role now